yaml/landuse.yaml

```diff
@@ -323,9 +326,6 @@ filters:
   - filter: {tourism: picnic_site}
     min_zoom: GREATEST(LEAST(zoom, 16), 9)
     output: {kind: picnic_site}
-  - filter: {historic: fort}
-    min_zoom: GREATEST(LEAST(zoom, 16), 9)
-    output: {kind: fort}
```

Comment:
I can't find any commits moving it. Is it possible that you pushed them to a branch which never got merged?